src/main.js

```diff
@@ -1,35 +1,40 @@
-import image from "../templates/images/star_filled.png";
-
-console.log("npm run dev 명령어를 통해 영화 리뷰 미션을 시작하세요");
-
-console.log(
-  "%c" +
-    " _____ ______   ________  ___      ___ ___  _______                \n" +
-    "|\\   _ \\  _   \\|\\   __  \\|\\  \\    /  /|\\  \\|\\  ___ \\               \n" +
-    "\\ \\  \\\\\\__\\ \\  \\ \\  \\|\\  \\ \\  \\  /  / | \\  \\ \\   __/|              \n" +
-    " \\ \\  \\\\|__| \\  \\ \\  \\\\\\  \\ \\  \\/  / / \\ \\  \\ \\  \\_|/__            \n" +
-    "  \\ \\  \\    \\ \\  \\ \\  \\\\\\  \\ \\    / /   \\ \\  \\ \\  \\_|\\ \\           \n" +
-    "   \\ \\__\\    \\ \\__\\ \\_______\\ \\__/ /     \\ \\__\\ \\_______\\          \n" +
-    "    \\|__|     \\|__|\\|_______|\\|__|/       \\|__|\\|_______|          \n" +
-    "                                                                   \n" +
-    "                                                                   \n" +
-    "                                                                   \n" +
-    " ________  _______   ___      ___ ___  _______   ___       __      \n" +
-    "|\\   __  \\|\\  ___ \\ |\\  \\    /  /|\\  \\|\\  ___ \\ |\\  \\     |\\  \\    \n" +
-    "\\ \\  \\|\\  \\ \\   __/|\\ \\  \\  /  / | \\  \\ \\   __/|\\ \\  \\    \\ \\  \\   \n" +
-    " \\ \\   _  _\\ \\  \\_|/_\\ \\  \\/  / / \\ \\  \\ \\  \\_|/_\\ \\  \\  __\\ \\  \\  \n" +
-    "  \\ \\  \\\\  \\\\ \\  \\_|\\ \\ \\    / /   \\ \\  \\ \\  \\_|\\ \\ \\  \\|\\__\\_\\  \\ \n" +
-    "   \\ \\__\\\\ _\\\\ \\_______\\ \\__/ /     \\ \\__\\ \\_______\\ \\____________\\\n" +
-    "    \\|__|\\|__|\\|_______|\\|__|/       \\|__|\\|_______|\\|____________|",
-  "color: #d81b60; font-size: 14px; font-weight: bold;"
-);
-
-addEventListener("load", () => {
-  const app = document.querySelector("#app");
-  const buttonImage = document.createElement("img");
-  buttonImage.src = image;
-
-  if (app) {
-    app.appendChild(buttonImage);
+import { createMovieList } from "./components/movie-list.js";
+import { createSkeleton, removeSkeleton } from "./components/skeleton.js";
+import { getPopularMovies } from "./services/movie-api.js";
+
+let currentPage = 1;
+
+addEventListener("load", async () => {
+  const thumbnailList = document.querySelector("main .thumbnail-list");
+  thumbnailList.appendChild(createSkeleton());
+
+  const popularMovieListData = await getPopularMovies();
+  await new Promise((resolve) => setTimeout(resolve, 500));
+  removeSkeleton();
+
+  const movieList = popularMovieListData.results;
+  currentPage = popularMovieListData.page;
+  const popularMovieList = createMovieList(movieList);
+
+  const moreButton = document.querySelector("main .more");
+  if (popularMovieListData.total_pages > currentPage) {
+    moreButton.classList.add("visible");
+  } else {
+    moreButton.classList.remove("visible");
   }
```

Comment:
if문 대신에 주어진 인수를 활용할 수 있는 [toggle](https://developer.mozilla.org/ko/docs/Web/API/Element/classList)로 활용해 보면 더욱 간소하고 명료하게 작성할 수 있답니다!

```suggestion
  moreButton.classList.toggle("visible", popularMovieListData.total_pages > currentPage);
```

또는 명료하게 두 줄로 추가하는 방법도 있습니다. 현석님께서 작성하시는 코드를 기준으로 명료하게 작성할 수 있는 방법을 골라 작성하시면 좋을 것 같아요~!

```js
  const hasMorePages = popularMovieListData.total_pages > currentPage;
  moreButton.classList.toggle("visible", hasMorePages);
```

Reply:
@sannimdev 
오 알려주신 코드가 훨씬 좋네요! 👍 
toggle을 활용하도록 하고, 추가로 초기 랜더링 시에는 className을 통해 지정하도록 변경하였습니다. 🙇 
32773a0c363022eef5119deb4613a92d300d5ba3